playbooks/vars/sample-variables-sap-vm-provision-redhat-ocpv.yml

```diff
@@ -3,54 +3,23 @@
 # Red Hat OpenShift Virtualization         #
 ############################################
 
+# kubeconfig for Red Hat OpenShift cluster connection.
+# If not provided, the kubeconfig will be read from the environment variables
+# KUBECONFIG or K8S_AUTH_KUBECONFIG
+# sap_vm_provision_kubevirt_vm_kubeconfig: /path/to/clusterconfigs/kubeconfig
+
 # Namespace where the VM should be created in
-sap_vm_provision_kubevirt_vm_target_namespace: sap
+sap_vm_provision_kubevirt_vm_target_namespace: openshift-sriov-network-operator
```

Comment:
we should avoid configuring vm in operator namespaces .

Reply:
True, but SRIOV has not been working outside of that namespace. But we can set the default to sap again and fix it in documentation.